website_form_recaptcha/tests/test_captcha.py

```diff
@@ -95,6 +95,43 @@ def test_no_success_raises(self, mk):
             self.model_obj.validate_response(
                 *self.validate_vars, website=self.website)
 
+    def test_validate_request_no_value(self):
+        request = object()
+        req_values = {}
+        with self.assertRaises(ValidationError) as err:
+            self.model_obj.validate_request(request, req_values)
+        self.assertEqual(
+            err.exception.name, 'The secret parameter is missing.')
+
+    def test_validate_request_old_value(self):
+        request = mock.MagicMock()
+        request.g_recaptcha_response = 'all good here'
+        req_values = {}
+        self.assertTrue(self.model_obj.validate_request(request, req_values))
```

Comment:
In this specific case, you probably need to ensure no calls to validation method are done, since the request is repeated.

It would be a bug if the validation is called when it shouldn't.